src/components/MyKiva/JournalUpdateCard.vue

```diff
@@ -123,6 +123,10 @@ const props = defineProps({
 		type: String,
 		default: '',
 	},
+	balance: {
```

Comment:
As mentioned in the other comments, this shouldn't be needed because of `livesToImpact`?